src/authentication.ts

```diff
@@ -115,14 +115,14 @@ async function getJwks(env: Env, securitySchemeName: SecuritySchemeName) {
   return jwksUrls[jwksUrl];
 }
 
-function isValidScopes(token: TokenData, scopes: string[]) {
-  if (!scopes.length) {
+function isValidScopes(token: TokenData, permissions: string[]) {
```

Comment:
```suggestion
function isValidPermissions(token: TokenData, permissions: string[]) {
```

Should we rename the function?


The mismatch of plurality between `is` and `scopes` always bothered me but it's not Polish :smile:

Reply:
Also of course commiting this will blow everything up